run-nim

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+. ./config
+
+if [ ! -f "spec/nim/test.nim" ]; then
+  RED='\033[1;31m'
+  YELLOW='\033[1;33m'
+  CYAN='\033[0;36m'
+  NC='\033[0m'
+
+  printf "${RED}Error:${NC} Test suite doesn't exist\n"
+  printf "${YELLOW}Hint:${NC} Must first run the ${CYAN}run-nim_generate_suite${NC} script\n"
+  exit
+fi
+
+rm -rf "$TEST_OUT_DIR/nim"
+mkdir -p "$TEST_OUT_DIR/nim/bin"
+
+i=0 args=()
+for var in "$@"
+do
+  args[$i]="Kaitai Struct Compiler Test Suite::Nim: $var"
```

Comment:
These are arrays, they won't work in /bin/sh. Either use /bin/bash, or use a more shell-like approach, such as processing arguments as individual lines with `sed 's/^/Kaitai Struct Compiler Test Suite::Nim: /'`

Reply:
I changed the shebang to /bin/bash for now, but if /bin/sh is preferred feel free to me.